rdr_client/hpo_assigner.py

```diff
@@ -54,20 +60,21 @@ def main(client):
         num_errors += 1
         continue
 
-      old_hpo = _get_old_hpo(participant)
-      if hpo == old_hpo:
+      old_pairing = _get_old_pairing(participant, pairing)
+      if new_pairing == old_pairing:
         num_no_change += 1
-        logging.info('%s unchanged (already %s)', participant_id, old_hpo)
+        logging.info('%s unchanged (already %s)', participant_id, old_pairing)
         continue
 
-      logging.info('%s %s => %s', participant_id, old_hpo, hpo)
-      if hpo == 'UNSET':
-        participant['providerLink'] = []
+      logging.info('%s %s => %s', participant_id, old_pairing, new_pairing)
+      if new_pairing == 'UNSET':
+        for i in pairing_list:
+          participant[i] = 'UNSET'
```

Comment:
Is it correct to set site and organization to UNSET? Or do we just null them out?

What about providerLink? Shouldn't we null that out?